validators/RequestSchema.js

```diff
@@ -11,6 +11,23 @@ const ItemSchema = FileSchema.keys({
   children: Joi.array().items(Joi.string()),
 })
 
+// Homepage
+const UpdateHomepageSchema = Joi.object({
+  content: Joi.object({
+    frontMatter: Joi.object({
+      layout: Joi.string().required(),
+      title: Joi.string().required(),
+      description: Joi.string().allow(""),
+      permalink: Joi.string().required(),
+      notification: Joi.string().allow(""),
+      image: Joi.string(),
```

Comment:
could i confirm that the frontend omit this property if empty?

Reply:
Yep, the image isn't modifiable inside EditHomepage - so if the image property is originally missing, the request will not contain image also